application/review/views.py

```diff
@@ -41,24 +49,22 @@ def review_page(review_token):
         abort(404)
 
 
-@review_blueprint.route("/new-review-url/<id>/<version>")
+@review_blueprint.route("/new-review-url/<id>")
 @login_required
-def get_new_review_url(id, version):
-    try:
-        token = generate_review_token(id, version)
-        measure_version = MeasureVersion.query.filter_by(guid=id, version=version).one()
-        measure_version.review_token = token
-        db.session.commit()
-        url = url_for("review.review_page", review_token=measure_version.review_token, _external=True)
-        expires = measure_version.review_token_expires_in(current_app.config)
-        day = "day" if expires == 1 else "days"
-        return """<a href="{url}">Review link</a> expires in {expires} {day}
-                    <button id="copy-to-clipboard" class="button neutral">Copy link</button>
-                    <input id="review-link" value="{url}">
-                    """.format(
-            expires=expires, day=day, url=url
-        )
+def get_new_review_url(id):
+    measure_version = MeasureVersion.query.filter_by(id=id).one()
+    if current_user.user_type == TypeOfUser.DEPT_USER and current_user not in measure_version.measure.shared_with:
```

Comment:
`current_user.can_access_measure(measure_version.measure)` might be the more canonical/DRY way to do this?

Reply:
Yeah good point, I hadn't pulled latest master so didn't have that until rebasing :man_facepalming: